Makefile

```diff
@@ -120,4 +121,5 @@ install_go_sdk::
 	#target intentionally blank
 
 install_nodejs_sdk::
+	yarn unlink --cwd $(WORKING_DIR)/sdk/nodejs/bin
```

Comment:
does this work if it's not already set? It usually failed for me

Reply:
Oops, good catch. Added a `-` prefix, which will ignore errors on that line